commit 851592d15ec1635314ca4fafb281d42f261f2a60
Author: Alexey Kardashevskiy <aik@ozlabs.ru>
Date:   Fri Mar 17 00:48:29 2017 +0000

    vfio/spapr: Postpone default window creation
    
    [ Upstream commit d9c728949ddc9de5734bf3b12ea906ca8a77f2a0 ]
    
    We are going to allow the userspace to configure container in
    one memory context and pass container fd to another so
    we are postponing memory allocations accounted against
    the locked memory limit. One of previous patches took care of
    it_userspace.
    
    At the moment we create the default DMA window when the first group is
    attached to a container; this is done for the userspace which is not
    DDW-aware but familiar with the SPAPR TCE IOMMU v2 in the part of memory
    pre-registration - such client expects the default DMA window to exist.
    
    This postpones the default DMA window allocation till one of
    the folliwing happens:
    1. first map/unmap request arrives;
    2. new window is requested;
    This adds noop for the case when the userspace requested removal
    of the default window which has not been created yet.
    
    Signed-off-by: Alexey Kardashevskiy <aik@ozlabs.ru>
    Reviewed-by: David Gibson <david@gibson.dropbear.id.au>
    Acked-by: Alex Williamson <alex.williamson@redhat.com>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1439_851592d15ec1635314ca4fafb281d42f261f2a60.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1439_851592d15ec1635314ca4fafb281d42f261f2a60.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
